### PR TITLE
Use symbol API to theme icon

### PR DESCRIPTION
**Pull request by NotMyFault**
Fixes [JENKINS-68422](https://issues.jenkins.io/browse/JENKINS-68422), by utilizing the symbol API rather then relying on the default color.

Current look:

Proposed change:

Closes #164

### CODE REVIEW

File: src/main/resources/jenkins/metrics/impl/TimeInQueueAction/summary.jelly [hunk 1]
```diff
@@ -22,9 +22,8 @@
  ~ THE SOFTWARE.
  -->
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout"
-         xmlns:t="/lib/hudson" xmlns:f="/lib/form">
-  <t:summary icon="/plugin/metrics/images/clock.svg" href="${it.urlName}" iconOnly="true">
+<j:jelly xmlns:j="jelly:core" xmlns:t="/lib/hudson" >
```
Comment (alecharp) [inline src/main/resources/jenkins/metrics/impl/TimeInQueueAction/summary.jelly:25]:
```suggestion
<j:jelly xmlns:j="jelly:core" xmlns:t="/lib/hudson">
```

**Review by alecharp — CHANGES_REQUESTED** [review]
The code looked ok, but when I run `mvn hpi:run` with an old run I get:

Screen Shot 2022-05-16 at 09 36 23

and had the same problem with a new run:

Screen Shot 2022-05-16 at 09 38 52

**Comment by alecharp** [comment]
I debugged the icon resolution and the `symbol-clock` is never in the `IconSet#iconsByCSSSelector`. Only when running with `2.347`

**Comment by alecharp** [comment]
(sorry, didn't mean to close the PR, clicked on the wrong button)

**Comment by NotMyFault** [comment]
> Only when running with 2.347, the icon is found.

Good catch, I forgot to bump the jenkins.version. Should be good now 👍🏻

**Comment by alecharp** [comment]
So, this is a good fix, but only for the latest weekly release. I would prefer not to require that on this plugin for the moment.

**Comment by NotMyFault** [comment]
If you prefer to keep it on an LTS line, we can use 2.346.1, once released.

**Comment by alecharp** [comment]
I would definitively prefer to wait for the LTS to be released.

**Comment by NotMyFault** [comment]
> I would definitively prefer to wait for the LTS to be released.

I've exchanged the jenkins.version with the weekly the upcoming LTS version is based on. Can increase it to .1 once released.

**Comment by NotMyFault** [comment]
I didn't forget you @alecharp :P

The plugin has been realigned with today's LTS release of 2.346.1

**Review by alecharp — APPROVED** [review]
Tested locally and couldn't find any problem.

**Comment by NotMyFault** [comment]
@alecharp anything left that blocks a merge and possible release?

**Comment by alecharp** [comment]
@NotMyFault nothing, except me. Fixing this now.

**Comment by jetersen** [comment]
This release is broken for support core, which refuses to load 4.1.6-383.v9d1dd57200e0 because it requires metrics-4.1.6.2 which is higher

https://github.com/jenkinsci/metrics-plugin/releases/tag/4.1.6-383.v9d1dd57200e0 is lower than https://github.com/jenkinsci/metrics-plugin/releases/tag/metrics-4.1.6.2

**Pull request merged**